--- mplay-0.42/install.orig	2004-04-12 17:31:46.895590368 +0200
+++ mplay-0.42/install	2004-04-12 17:42:19.608403400 +0200
@@ -1,132 +1,44 @@
 #! /bin/sh
 
-if ! [ $HOME = /root ]
-	then echo -e "\e[31mYou must become root to run \e[32minstall\e[0m\nAborting";exit
-fi 
-type perl &> /dev/null
-if ! [ $? = 0 ]
-	then echo -e "\e[31mI can't find Perl in your PATH\e[0m\nAborting";exit
-fi
-ln -s `type -p perl` /usr/bin/perl &> /dev/null
-type mplayer &> /dev/null
-if ! [ $? = 0 ]
-	then echo -e "\e[31mI can't find MPlayer in your PATH\e[0m\nAborting\n\nRead README where you can get MPlayer";exit
-fi 
-type make &> /dev/null
-if ! [ $? = 0 ]
-	then echo -e "\e[31mI can't find make in your PATH\e[0m\nAborting";exit
-fi 
-echo -e "\n\e[1m  Starting ....\e[0m";
-perl -e 'use Audio::Mixer ;' &> /dev/null ;mixer=$?
-perl -e 'use MP3::Info ;' &> /dev/null ;mp3=$?
-perl -e 'use Term::ReadKey ;' &> /dev/null ;readkey=$?
-perl -e 'use Time::HiRes ;' &> /dev/null ;hires=$?
-perl -e 'use File::PathConvert ;' &> /dev/null ;path=$?
-perl -e 'use Term::Screen ;' &> /dev/null ;screen=$?
-echo
-if ! [ $mixer = 0 ]
-	then echo -e "\e[32m Audio::Mixer\e[0m"
-fi 
-if ! [ $mp3 = 0 ]
-	then echo -e "\e[32m MP3::Info\e[0m"
-fi 
-if ! [ $readkey = 0 ]
-	then echo -e "\e[32m Term::ReadKey\e[0m"
-fi 
-if ! [ $hires = 0 ]
-	then echo -e "\e[32m Time::Hires\e[0m"
-fi 
-if ! [ $path = 0 ]
-	then echo -e "\e[32m File::PathConvert\e[0m"
-fi 
-if ! [ $path = 0 ]
-	then echo -e "\e[32m Term::Screen\e[0m"
-fi 
-
-#if [ $mixer = 0 || $mp3 = 0 || 
-tmp=0;
-if ! [ $mixer = 0 ]
-	then tmp=1
-	elif ! [ $mp3 = 0 ]
-	then tmp=1
-	elif ! [ $readkey = 0 ]
-	then tmp=1
-	elif ! [ $hires = 0 ]
-	then tmp=1
-	elif ! [ $path = 0 ]
-	then tmp=1
-	elif ! [ $screen = 0 ]
-	then tmp=1
-fi
-if [ $tmp = 1 ]
-	then echo -e -n "\n The perl-module(s) above have to be installed.\n Press \e[32mEnter\e[0m to proceed or \e[31mCtrl-C\e[0m to abort.\r"
-	read
-fi
-
-if ! [ $mixer = 0 ]
-	then 	tar -xzf src/Audio-Mixer-0.7.tar.gz
-		cd Audio-Mixer-0.7/
-		perl Makefile.PL
-		make
-		make install
-		cd ..
-		rm -r -f Audio-Mixer-0.7/
-fi 
-if ! [ $mp3 = 0 ]
-	then 	tar -xzf src/MP3-Info-1.02.tar.gz
-		cd MP3-Info-1.02/
-		perl Makefile.PL
-		make
-		make install
-		cd ..
-		rm -r -f MP3-Info-1.02/
-fi 
-if ! [ $readkey = 0 ]
-	then 	tar -xzf src/TermReadKey-2.21.tar.gz
-		cd TermReadKey-2.21/
-		perl Makefile.PL
-		make
-		make install
-		cd ..
-		rm -r -f TermReadKey-2.21/
-fi 
-if ! [ $hires = 0 ]
-	then 	tar -xzf src/Time-HiRes-1.56.tar.gz
-		cd Time-HiRes-1.56/
-		perl Makefile.PL 
-		make
-		make install
-		cd ..
-		rm -r -f Time-HiRes-1.56/
-fi 
-if ! [ $path = 0 ]
-	then 	tar -xzf src/File-PathConvert-0.85.tar.gz
-		cd File-PathConvert-0.85/
-		perl Makefile.PL 
-		make
-		make install
-		cd ..
-		rm -r -f File-PathConvert-0.85/
-fi 
-if ! [ $screen = 0 ]
-	then 	tar -xzf src/Term-Screen-1.02.tar.gz
-		cd Term-Screen-1.02/
-		perl Makefile.PL 
-		make
-		make install
-		cd ..
-		rm -r -f Term-Screen-1.02/
-fi 
-
-
-if  ! [ -d ~/.mplay ]
-	then mkdir ~/.mplay
-fi
-if  ! [ -d /usr/local/share/mplay ]
-	then mkdir -p /usr/local/share/mplay
-fi
-
-echo -e "\nCopy help files to /usr/local/share/mplay/";cp help_en help_de /usr/local/share/mplay/
-echo "Copy mplay to /usr/local/bin/";cp mplay /usr/local/bin/
-chmod +x /usr/local/bin/mplay
-echo -e "\n\e[32mReady\n\e[0m\nNow you can run e.g.: \e[33mmplay /path/to/some/*.mp3\e[0m\nAnd then press \e[31mh\e[0m to see the help.\nHave fun ...\n"
+tar -xzf src/Audio-Mixer-0.7.tar.gz
+cd Audio-Mixer-0.7/
+perl Makefile.PL INSTALLDIRS=vendor
+make
+make install DESTDIR=$RPM_BUILD_ROOT
+cd ..
+rm -r -f Audio-Mixer-0.7/
+tar -xzf src/MP3-Info-1.02.tar.gz
+cd MP3-Info-1.02/
+perl Makefile.PL INSTALLDIRS=vendor
+make
+make install DESTDIR=$RPM_BUILD_ROOT
+cd ..
+rm -r -f MP3-Info-1.02/
+tar -xzf src/TermReadKey-2.21.tar.gz
+cd TermReadKey-2.21/
+perl Makefile.PL INSTALLDIRS=vendor
+make
+make install DESTDIR=$RPM_BUILD_ROOT
+cd ..
+rm -r -f TermReadKey-2.21/
+tar -xzf src/Time-HiRes-1.56.tar.gz
+cd Time-HiRes-1.56/
+perl Makefile.PL INSTALLDIRS=vendor
+make
+make install DESTDIR=$RPM_BUILD_ROOT
+cd ..
+rm -r -f Time-HiRes-1.56/
+tar -xzf src/File-PathConvert-0.85.tar.gz
+cd File-PathConvert-0.85/
+perl Makefile.PL INSTALLDIRS=vendor
+make
+make install DESTDIR=$RPM_BUILD_ROOT
+cd ..
+rm -r -f File-PathConvert-0.85/
+tar -xzf src/Term-Screen-1.02.tar.gz
+cd Term-Screen-1.02/
+perl Makefile.PL INSTALLDIRS=vendor
+make
+make install DESTDIR=$RPM_BUILD_ROOT
+cd ..
+rm -r -f Term-Screen-1.02/
